OneBusAway/ui/themes/OBAClassicApplicationUI.m

```diff
@@ -82,6 +82,11 @@ - (UIViewController*)rootViewController {
     return self.tabBarController;
 }
 
+- (void)pushViewController:(UIViewController *)vc {
```

Comment:
This should be removed and replaced with code (in the app delegate) that calls into `-[OBAApplicationDelegate navigateToTarget:]`. Also, can you talk a little bit about why you think this should be pushed from the map? I'm not opposed to it; I just want to hear your reasoning.